tfhe/src/integer/server_key/radix_parallel/tests_unsigned/test_sub.rs

```diff
@@ -62,14 +44,6 @@ where
     default_sub_test(param, executor);
 }
 
-fn integer_default_sub_work_efficient<P>(param: P)
```

Comment:
Yes because it was slower than the previous "low_latency" addition, and from the throughput benches I did a while ago, the window where the work_efficient algorithm had a better throughput than the sequential and low_latency was small.

And given that the new algorithm uses less threads, I expects the thoughput to be better, thus the windows for which the work_efficient has the best troughput should be smaller